keyboards/newgame40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,213 @@
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   -  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |Shift |  GUI |Lower | Ctrl |Space | Bksp |Enter |Raise | Left |Right |
+  * `---------------------------------------------------------------------'
+  */
+ [_QWERTY] = LAYOUT( \
+   KC_Q,     KC_W,     KC_E,      KC_R,      KC_T,      KC_Y,      KC_U,      KC_I,      KC_O,      KC_P,     \
+   KC_A,     KC_S,     KC_D,      KC_F,      KC_G,      KC_H,      KC_J,      KC_K,      KC_L,      KC_MINS,  \
+   KC_Z,     KC_X,     KC_C,      KC_V,      KC_B,      KC_N,      KC_M,      KC_COMM,   KC_DOT,    KC_SLSH,  \
+   KC_LSFT, KC_LGUI, LT(_LOWER,KC_LANG2),  KC_LCTL, KC_SPC,  KC_BSPC, KC_ENT, LT(_RAISE,KC_LANG1),  KC_LEFT, KC_RGHT\
+ ),
+
+ /* Colemak
+  * ,---------------------------------------------------------------------.
+  * |   Q  |   W  |   F  |   P  |   G  |   J  |   L  |   U  |   Y  |   -  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   A  |   R  |   S  |   T  |   D  |   H  |   N  |   E  |   I  |   O  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   Z  |   X  |   C  |   V  |   B  |   K  |   M  |   ,  |   .  |   /  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |Shift |  GUI |Lower | Ctrl |Space | Bksp |Enter |Raise | Left |Right |
+  * `---------------------------------------------------------------------'
+  */
+ [_COLEMAK] = LAYOUT( \
+   KC_Q,     KC_W,     KC_F,      KC_P,      KC_G,      KC_J,      KC_L,      KC_U,      KC_Y,      KC_MINS,  \
+   KC_A,     KC_R,     KC_S,      KC_T,      KC_D,      KC_H,      KC_N,      KC_E,      KC_I,      KC_O,     \
+   KC_Z,     KC_X,     KC_C,      KC_V,      KC_B,      KC_K,      KC_M,      KC_COMM,   KC_DOT,    KC_SLSH,  \
+   KC_LSFT, KC_LGUI, LT(_LOWER,KC_LANG2),  KC_LCTL, KC_SPC,  KC_BSPC, KC_ENT, LT(_RAISE,KC_LANG1),  KC_LEFT, KC_RGHT\
+ ),
+
+ /* Dvorak
+  * ,---------------------------------------------------------------------.
+  * |   "  |   ,  |   .  |   P  |   Y  |   F  |   G  |   C  |   R  |   L  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   A  |   O  |   E  |   U  |   I  |   D  |   H  |   T  |   N  |   S  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   ;  |   Q  |   J  |   K  |   X  |   B  |   M  |   W  |   V  |   Z  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |Shift |  GUI |Lower | Ctrl |Space | Bksp |Enter |Raise | Left |Right |
+  * `---------------------------------------------------------------------'
+  */
+ [_DVORAK] = LAYOUT( \
```

Comment:
If you're using `LAYOUTS = ortho_4x10`, then you should use that here: 

```suggestion
 [_DVORAK] = LAYOUT_ortho_4x10( \
```